### PR TITLE
Milestone 1: Unit test for Add and delete function

### PR DESCRIPTION
**Pull request by Rachelwebdev**
### In this milestone, we:
- Installed Jest Framework for testing
- Created a test file containing two tests for add and delete item functions
- Used Mocks to manipulate the DOM and imitate the local storage.
- Used the describe method to group tests
- Fixed linter errors

### CODE REVIEW

File: src/modules/listCard.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 
-const toDoList = document.querySelector(".toDoContainer");
+export const toDoList = document.querySelector('.toDoContainer');
 
 export const displayListTasks = (tasksArray) => {
-  toDoList.innerHTML = "";
+  toDoList.innerHTML = '';
 
   tasksArray = getLocalStorage();
   tasksArray.forEach((toDo) => {
```
Comment (Munsa1) [inline src/modules/listCard.js]: - [ ] One test is failing, kindly fix this failing test as this is a requirement.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev  @Lancelot-SO, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/listCard.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/modules/listCard.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Rachelwebdev   @Lancelot-SO 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git -Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ All Test Are Passing.
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rachelwebdev** [comment]
Thanks for the approval 👍

**Pull request merged**